### PR TITLE
[MM-49587] Fixing layout issue when banner appears

### PR DESCRIPTION
**Pull request by BenCookie95**
#### Summary
Fixing layout issue when banner appears on the following pages: 
/login
/access_problem
/signup_user_complete
/should_verify_email
/do_verify_email

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49587

#### Related PRs
https://github.com/mattermost/mattermost-webapp/pull/11920

#### Screenshots
Screen Shot 2023-01-13 at 11 50 47 AM

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by kostaspt — CHANGES_REQUESTED** [review]
It looks like it's still broken for smaller screens.

https://user-images.githubusercontent.com/684680/212394893-f4fd3bbd-35fa-4a12-bb2b-4f57d5214919.mp4

**Comment by BenCookie95** [comment]
> It looks like it's still broken for smaller screens.
> 
>  Kapture.2023-01-13.at.20.40.03.mp4

Nice find!

**Comment by BenCookie95** [comment]
> It looks like it's still broken for smaller screens.
> 
>  Kapture.2023-01-13.at.20.40.03.mp4

Fixed, the announcement banner also completely broke the mobile view of channels too

**Review by kostaspt — APPROVED** [review]
Looks good now 👍🏼

**Comment by kostaspt** [comment]
This probably fixes more issues, for example: https://mattermost.atlassian.net/browse/MM-49501

File: sass/base/_structure.scss [hunk 1]
```diff
@@ -159,6 +159,10 @@ body.app__body #root {
             position: relative;
             left: 65px;
         }
+
+        .announcement-bar {
+            grid-area: header;
+        }
```
Comment (calebroseland) [inline sass/base/_structure.scss:165]: This should be fixed once #11920 is merged.

```suggestion
```

**Review by calebroseland — CHANGES_REQUESTED** [review]
Looks right for `header-footer-route` 👍.  Just one duplicate on the mobile grid part.

**Review by calebroseland — APPROVED** [review]

**Comment by BenCookie95** [comment]
@amyblais @furqanmlk Closing this PR as it was already fixed here https://github.com/mattermost/mattermost-webapp/pull/11867

**Pull request closed without merging**